test/colony-network-staking.js

```diff
@@ -682,30 +682,30 @@ contract("ColonyNetworkStaking", accounts => {
       await colonyNetwork.deposit("1000000000000000000", { from: accounts[2] });
 
       const addr = await colonyNetwork.getReputationMiningCycle.call();
-      await testHelper.forwardTime(3600, this);
+      await forwardTime(3600, this);
       const repCycle = ReputationMiningCycle.at(addr);
       await repCycle.submitNewHash("0x12345678", 10, 10);
       await repCycle.submitNewHash("0x87654321", 11, 10, { from: OTHER_ACCOUNT });
       await repCycle.submitNewHash("0x99999999", 12, 10, { from: accounts[2] });
       await accommodateChallengeAndInvalidateHash(this, 0, 1);
       await accommodateChallengeAndInvalidateHash(this, 0, 3, false); // Invalidate the 'null' that partners the third hash submitted
       // No response to a challenge required.
-      await testHelper.checkErrorRevert(accommodateChallengeAndInvalidateHash(this, 1, 2, false));
+      await checkErrorRevert(accommodateChallengeAndInvalidateHash(this, 1, 2, false));
       // Cleanup after test
       await accommodateChallengeAndInvalidateHash(this, 1, 0);
       await repCycle.confirmNewHash(2);
     });
 
     it("should keep reputation updates that occur during one update window for the next window", async () => {
-      await testDataGenerator.giveUserCLNYTokens(colonyNetwork, MAIN_ACCOUNT, "1000000000000000000");
+      await giveUserCLNYTokens(colonyNetwork, MAIN_ACCOUNT, "1000000000000000000");
       await clny.approve(colonyNetwork.address, "1000000000000000000");
       await colonyNetwork.deposit("1000000000000000000");
       const addr = await colonyNetwork.getReputationMiningCycle.call();
-      await testHelper.forwardTime(3600, this);
+      await forwardTime(3600, this);
       const repCycle = ReputationMiningCycle.at(addr);
       await repCycle.submitNewHash("0x12345678", 10, 10);
-      await testDataGenerator.fundColonyWithTokens(commonColony, clny, "300000000000000000000");
-      const taskId1 = await testDataGenerator.setupRatedTask(colonyNetwork, commonColony);
+      await fundColonyWithTokens(commonColony, clny, "350000000000000000000");
```

Comment:
Why did this value change? Not that it makes a difference in the test but wondering?

Reply:
The test fails on [this](https://github.com/JoinColony/colonyNetwork/blob/develop/helpers/test-data-generator.js#L119) line i think, since there are not enough tokens to move. I don't know how this was not failing earlier. Maybe i'm missing something

Reply:
Just debugged this to see what's happening and found it's due to the unit test actually inheriting commonColony funds from previous test. The totalPayouts are correctly `350000000000000000000`. 
Since the common colony funding is not under testing here and tests are very slow already, I'm inclined to accept a little dirty test harness setup.